src/openforms/analytics_tools/tests/test_analytics_rendering.py

```diff
@@ -1,13 +1,15 @@
+from django.test import override_settings
 from django.urls import reverse
 
 from cookie_consent.models import CookieGroup
 from django_webtest import WebTest
 
 from openforms.analytics_tools.models import AnalyticsToolsConfiguration
 from openforms.forms.tests.factories import FormFactory
-from openforms.utils.tests.cache import clear_caches
+from openforms.tests.utils import NOOP_CACHES
 
 
+@override_settings(CACHES=NOOP_CACHES)
```

Comment:
Do we use this for CI? I see that with or without it the tests are passing locally.

Reply:
without it, there's a risk that other tests are affected because the cache is shared between tests. So it's to prevent future issues and ensure test isolation.